packages/orchestrator-ui-components/src/components/WFOSubscription/WFOSubscriptionDetailTree.tsx

```diff
@@ -22,36 +22,42 @@ import { WFOSubscriptionProductBlock } from './WFOSubscriptionProductBlock';
 
 type NodeMap = { [key: number]: TreeBlock };
 
-const MAX_EXPAND_ALL = 100;
-
 interface WFOSubscriptionDetailTreeProps {
     productBlockInstances: ProductBlockInstance[];
 }
 
+function getNodeDepth(node: TreeBlock, idToNodeMap: NodeMap): number {
```

Comment:
could you add some unit tests for this one?

Reply:
yes